raiden-dapp/src/components/transfer/TransferMenus.vue

```diff
@@ -105,6 +105,7 @@ export default class TransferMenus extends Mixins(NavigationMixin) {
     return this.capacity === Zero;
   }
 
+  /* istanbul ignore next */
```

Comment:
Why don't we test that? Shouldn't this get executed? Feels bad to ignore this here...

Reply:
I believe it is implicitly getting tested here?

```
  test('displays "no open channels" if channel capacity is zero', () => {
    const wrapper = createWrapper(Zero);
    const amountDisplay = wrapper.findAll('span').at(3);

    expect(amountDisplay.text()).toContain(
      'transfer.transfer-menus.no-channels'
    );
  });
```

Reply:
Ah wait this is the logic for the other component right? So this will be part of the refactoring of the deposit dialog?

Reply:
Ah, yes. I thought you were referring to: `return this.capacity === Zero;` :)